README.md

```diff
@@ -2,28 +2,27 @@
 
 # LXD
 
-<!-- Include start LXD intro -->
-
 LXD is a modern, secure and powerful system container and virtual machine manager.
 
+<!-- Include start LXD intro -->
+
 It provides a unified experience for running and managing full Linux systems inside containers or virtual machines. LXD supplies images for a [wide number of Linux distributions](https://images.linuxcontainers.org) and is built around a very powerful, yet pretty simple, REST API. LXD scales from one instance on a single machine to a cluster in a full data center rack, making it suitable for running workloads both for development and in production.
 
 LXD allows you to easily set up a system that feels like a small private cloud. You can run any type of workload in an efficient way while keeping your resources optimized.
 
 You should consider using LXD if you want to containerize different environments or run virtual machines, or in general run and manage your infrastructure in a cost-effective way.
 
-## Get started
 
-To get a better idea of what LXD is and what it does, you can [try it online](https://linuxcontainers.org/lxd/try-it/)!
-This interactive demo guides you through the basic steps of using LXD and shows how to launch and inspect your first instances, update the configuration, work with commands and files inside an instance and create snapshots.
+<!-- Include end LXD intro -->
+
+## Get started
 
-Then, if you want to run LXD locally, take a look at our [getting started guide](https://linuxcontainers.org/lxd/getting-started-cli/) for installation instructions.
+See [Getting started](https://linuxcontainers.org/lxd/docs/latest/getting_started/) in the LXD documentation for installation instructions and first steps.
```

Comment:
Is it possible to make this link a relative one (now its inside the docs itself) or to use a sphinx reference somehow so it generates a link to the relevant page (sorry I do not know the correct sphinx terminology).

Reply:
This is the readme file of the repo - so this section is not included in the actual documentation output. 
We could change it to a relative link, but then it would link to the Markdown file in the Git repo and not to the documentation output, and that's not how we want users to read the documentation.

Reply:
Thanks this makes perfect sense now :)